website/docs/reference/commands/list.md

```diff
@@ -93,6 +93,16 @@ $ dbt ls --select snowplow.* --output json --output-keys name resource_type desc
 
 </VersionBlock>
 
+<VersionBlock firstVersion="1.6">
+
+**Listing Semantic models**
+
+```
+$ dbt ls -s +semantic_model:number                      
+
```

Comment:
@dave-connors-3 offered this suggestion as a more clear example:
```suggestion
# list all resources upstream of your orders semantic model
dbt ls -s +semantic_model:orders
```

The key insight is that `number` in the PR description for https://github.com/dbt-labs/dbt-core/pull/8589 was meant to be the _name_ of a semantic model (rather than an indication of a numeric quantity).

Reply:
Thank you!